internal/generator/vector/conf/complex.toml

```diff
@@ -29,6 +29,7 @@ type = "remap"
 inputs = ["input_audit_host"]
 source = '''
   . = {"_internal": .}
+  if exists(._internal.stream) {._internal.kubernetes.container_iostream = ._internal.stream}
```

Comment:
This will never be a value from audit sources since they are file sources and kubernetes.   Can this be "removed" without  a large change?  If not, then I'm good with leaving it.

Reply:
That is why I added the `exists()` since they won't have a value and will not set that value. I think I can add a condition however. I will play around with it.